db-init/init.sql

```diff
@@ -1,2 +0,0 @@
-CREATE DATABASE IF NOT EXISTS mapbot;
-USE mapbot;
```

Comment:
`init.sql` script not needed for PostgreSQL. `docker` setup seems to be working fine without it.

The new database is created by `POSTGRES_DB` variable in the environment of the `mapbot_db` container.